### ISSUE TITLE
Logits and scores of semantic prediction

### ISSUE THREAD

**Issue by sonukiller** [google-research/deeplab2#154]
The model output for the semantic predictions is in the format:

semantic logits : (batch, 81, 81, num_classes) (For a crop_size of 321)
semantic scores : (batch, img_height, img_width, num_classes).

How to get the semantic logits in the exact shape of semantic scores?
